### PR TITLE
US 045 dashboard for admin

### PR DESCRIPTION
**Pull request by rocioferreiro**
## Description

> Dashboard has relevant information for the admin user. AmountComponent shows numeric information, such as delayed loans, and total amount of books. ChartComponent is used to display information in the form of a chart, right now only takes line charts or doughnuts. The rest of the information is displayed in tables.
## ClubHouse Link

> https://app.clubhouse.io/bibliotecame/story/303/dashboard-admin
## Checklist
- [X] I've manually tested the feature and it works as expected
- [X] I've successfully integrated the feature with the API
- [X] This pull request has 'sprint_x' as the base branch
- [X] This pull request has a descriptive title and description
- [X] I have merged the current sprint into my branch before pushing
- [X] I've added the QA Leader as a reviewer

### CODE REVIEW

**Review by FacundoBocalandro — CHANGES_REQUESTED** [review]
Revisar layout del contenedor

Screenshot from 2020-11-07 12-04-19

File: src/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        getInfo();
+    }, [])
+
+    const getInfo = () => {
+        get('dashboard').then(res => {
+            setInfo(res)
+        })
+    }
+
+    const reduce = (key : string) => {
```
Comment (FacundoBocalandro) [inline src/Dashboard/Dashboard.tsx]: Modificar nombre, por ejemplo abbreviateDate(...)

File: src/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+type Book = {
+    title: string,
+    author: string,
+    avgScore: number,
+    amountOfLoans: number
+}
+
+type MapEntry = {
```
Comment (FacundoBocalandro) [inline src/Dashboard/Dashboard.tsx]: Modificar nombre para que sea más específico, por ejemplo DateTuple

File: src/Dashboard/ChartComponent/ChartComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import "./ChartComponent.css"
+import {Line, Doughnut} from 'react-chartjs-2';
+
+type Props = {
+    type: string,
+    data: any,
+    options?: any,
```
Comment (FacundoBocalandro) [inline src/Dashboard/ChartComponent/ChartComponent.tsx:9]: Intercalar con el de abajo

File: src/Dashboard/Table/BestReviewsTable.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import GenericTable, {Column} from "../../common/GenericTable/GenericTable";
+import Rating from "react-rating";
+import './tables.css'
+
+type Props = {
+    data: any[],
```
Comment (FacundoBocalandro) [inline src/Dashboard/Table/BestReviewsTable.tsx]: Cambiar por Book[]

File: src/Dashboard/Table/MostLoanedTable.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import GenericTable, {Column} from "../../common/GenericTable/GenericTable";
+import './tables.css'
+type Props = {
+    data: any[],
```
Comment (FacundoBocalandro) [inline src/Dashboard/Table/MostLoanedTable.tsx]: Cambiar por Book[]

File: src/Dashboard/Dashboard.css [hunk 6]
```diff
@@ -0,0 +1,25 @@
+.dashboard{
+    background-color: var(--very-light-pink);
+    overflow-y: scroll;
+    max-height: 700px;
```
Comment (FacundoBocalandro) [inline src/Dashboard/Dashboard.css]: Cambiar a viewport

File: src/Dashboard/Table/tables.css [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    border-radius: 33px;
+    background-color: var(--white);
+    box-shadow: 10px 10px 6px 0 rgba(0, 0, 0, 0.16);
+}
+
+.dashboard-table .title{
+    background-color: var(--dodger-blue);
+    border-radius: 33px 33px 0 0;
+    align-items: center;
+    height: 5rem;
```
Comment (FacundoBocalandro) [inline src/Dashboard/Table/tables.css]: Cambiar a viewport

**Review by FacundoBocalandro — CHANGES_REQUESTED** [review]
Revisar

**Comment by rocioferreiro** [comment]
@FacundoBocalandro ahi debería estar

**Review by FacundoBocalandro — APPROVED** [review]
LGTM

**Review by LucasManza — CHANGES_REQUESTED** [review]
Ver si es posible habilitar la animación del grafico de lineas
image
Ref: https://www.chartjs.org/samples/latest/scales/logarithmic/line.html

**Comment by LucasManza** [comment]
Revisar Layout  --> Resolución 1920x1080
image

**Comment by rocioferreiro** [comment]
@LucasManza listo los cambios

**Review by LucasManza — APPROVED** [review]

**Pull request merged**